CHANGES.md

```diff
@@ -1,3 +1,61 @@
+# 0.9.2
+
+This minor release contains various bug fixes and improvements.
+
+## TODO: Bug breaking web examples
```

Comment:
```suggestion
## Breaking change: Trailing semicolons are no longer permitted

The surface syntax of Links has been changed. Up until now it was possible to end a block with a semicolon. A trailing semicolon was interpreted as implicitly ending the block with a `()` expression.  The rationale for this change is to make the Links syntax more consistent, i.e. now all blocks must end with an explicit expression. 

To sum up, previously both of the following were allowed

```links
fun foo(x) {
  bar(y);
  baz(x);
}

fun foo'(x) {
  bar(y);
  baz(x);
}
```

Now the first form is no longer accepted. Instead you have to drop the semicolon and either end the block with an explicit `()` or wrap the last expression in an `ignore` application.

```links
fun foo(x) {
  bar(y);
  baz(x); 
  ()
}

fun foo(x) {
  bar(y);
  ignore(baz(x))
}
```

A third option is to simply drop the trailing semicolon, though, this only works as intended if the type of the last expression is `()`.

## TODO: Bug breaking web examples
```

Reply:
It seems like the Github UI have cropped my suggestion, you may have to "edit" my comment and copy-paste the source code if you want to adopt it.

Reply:
I assume that in `foo'` (after "To sum up, previously both of the following were allowed"), you meant to omit the second `;`?

Reply:
Absolutely!